poetry.lock

```diff
@@ -2806,131 +2807,131 @@ files = [
 
 [[package]]
 name = "pydantic"
-version = "2.0.3"
+version = "2.1.1"
```

Comment:
Not too sure about this update... tests will tell.

Reply:
Meh, seems mostly fixes: https://docs.pydantic.dev/latest/changelog/#v210-2023-07-25